documentation/kubernetes-upgrade.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Upgrading kubeadm clusters from v1.7/v1.8 to v1.9 with vSphere Cloud Provider
+---
+
+## Before you begin
+
+Before proceeding:
+
+* You need to have a functional Kubernetes cluster deploed using kubeadm running version 1.7.0 or higher with vSphere Cloud Provider configured in order to use the process described here. If vSphere Cloud Provider in not enabled in your Kubernetes cluster, you can follow the instructions mentioned in this [link] (https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/existing.html#manual-steps-if-needed-to-enable-vsphere-cloud-provider) to manually enable vSphere Cloud Provider.
+* Make sure swap is disabled.
```

Comment:
Can you also describe the step to disable it?

For my clarify, can you explain a bit more as to why this is needed?